configs/components/cpp-pcp-client.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "2a0cbb5dbab8c967272b9cd04a0a923b6b42b73f"}
+{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "refs/tags/1.1.2"}
```

Comment:
which was handled in https://github.com/puppetlabs/puppet-agent/pull/646/commits/4116dfaa22a95abd246822e8e87fe3beecf81040